workers/unity/Packages/com.improbable.gdk.guns/Behaviours/FirstPersonRecoil.cs

```diff
@@ -39,6 +39,11 @@ private static Vector3 CalculateNewVelocity(GunSettings.FirstPersonRecoilSetting
 
         private void LateUpdate()
         {
+            if (aimRecoil == null || hipRecoil == null)
```

Comment:
What is this change for?

Reply:
As said in the description this is to deal with an ordering change made in the new core stuff. Previously this was assuming the monobehaviour would enable before another, which was never garunteed.